SpecializedServers/Horizon/SERVER/Medius/MAPS.cs

```diff
@@ -84,8 +84,13 @@ protected override async Task ProcessMessage(BaseScertMessage message, IChannel
                         }
 
                         data.ClientObject = MediusClass.Manager.GetClientByAccessToken(clientConnectTcp.AccessToken, clientConnectTcp.AppId);
```

Comment:
There is a problem here, the token client fetcher is present 2 times, if you need a new client object copy the MAS method.